spec/requests/orders/show_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe "Show Order" do
+    let(:product) { FactoryBot.create(:product, :with_image, price: 120) }
+    let(:user) { FactoryBot.create(:user) }
+    let!(:cart) { FactoryBot.create(:cart, user: user) }
+    let(:params) do
```

Comment:
params are only needed if you pass them in the request, otherwise they are not necessary

Reply:
same for other tests, the only requests that uses params are the create and the update, as you are setting new values.